employee/models.py

```diff
@@ -29,16 +29,16 @@ class Employee(models.Model):
     dob = models.DateField(verbose_name='Date of Birth')
     gender = models.IntegerField(choices=GENDER)
     email = models.EmailField(max_length=255, unique=True)
-    phone_number = models.PositiveIntegerField()
+    phone_number = models.BigIntegerField()
     address = models.CharField(max_length=255)
     pan_no = models.CharField(max_length=255, unique=True)
     citizenship_no = models.CharField(max_length=255, unique=True)
     designation = models.ForeignKey(
         Designation, on_delete=models.DO_NOTHING)
     join_date = models.DateField()
-    picture = models.ImageField(upload_to='pictures/')
-    pan_no_document = models.ImageField(upload_to='pictures/')
-    cititzen_document = models.ImageField(upload_to='pictures/')
+    picture = models.ImageField(upload_to='images/')
+    pan_no_document = models.ImageField(upload_to='images/')
+    cititzen_document = models.ImageField(upload_to='images/')
```

Comment:
```suggestion
    picture = models.ImageField(upload_to='images/pictures/')
    pan_no_document = models.ImageField(upload_to='images/pans/')
    cititzen_document = models.ImageField(upload_to='images/citizenships/')
```
This will make these documents more accessible to normal users looking into the `images` directory.